about.html

```diff
@@ -173,58 +179,57 @@
+                                <span class="cmp-button__icon cmp-button__icon--facebook facebook" aria-hidden="true">
+                                  <img src="./assets/facebook.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
+                                </span>
                               </a>
                             </div>
-                          </div>
-
-                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
-                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_512298232.coreimg{.width}.svg/1669130210337/insta.svg" data-asset="/content/dam/LeaderImpact/site-./assets/insta.svg" data-title="instagram" id="image-ae3671446d" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
-                              <a href="#" class="cmp-image__link">
-                                <img src="./assets/insta.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="instagram">
+                            <div class="button image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                              <a href="#" class="cmp-button">
+                                <span class="cmp-button__icon cmp-button__icon--instagram instagram" aria-hidden="true">
+                                  <img src="./assets/insta.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
+                                </span>
                               </a>
                             </div>
-                          </div>
-
-                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
-                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1887433809.coreimg{.width}.svg/1669130230195/yt.svg" data-asset="/content/dam/LeaderImpact/site-./assets/yt.svg" data-title="YouTube" id="image-46d1bf6845" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
-                              <a href="#" class="cmp-image__link">
-                                <img src="./assets/yt.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="YouTube">
+                            <div class="button image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                              <a href="#" class="cmp-button">
+                                <span class="cmp-button__icon cmp-button__icon--youtube youtube" aria-hidden="true">
+                                  <img src="./assets/yt.svg" class="cmp-image-button__icon" itemprop="contentUrl" data-cmp-hook-image="image" alt="Close">
```

Comment:
@Prystal254 Button components don't use `<img />` for the icons. You need to use a pseudo element or background image.